src/UI/Window.cpp

```diff
@@ -766,3 +766,10 @@ void cWindow::SetProperty(short a_Property, short a_Value, cPlayer & a_Player)
 
 
 
+
+
+void cWindow::LoadRecipe(cPlayer & a_Player, UInt32 a_RecipeId)
+{
+	auto slotAreaCrafting = static_cast<cSlotAreaCrafting *>(m_SlotAreas[0]);
```

Comment:
Not all windows have a `cSlotAreaCrafting`, so this doesn't belong in `cWindow`.